lerna.json

```
@@ -1,4 +1,5 @@
 {
+  "useNX": false,
```

Requires a lowercase "x":

```suggestion
  "useNx": false,
```